src/styles_replace-icon-with-svg.scss

```diff
@@ -0,0 +1,27 @@
+@import "./styles";
+@import "~@backbase/backbase-theme/scss/main";
+
+// Overriding the icon
```

Comment:
I would prefer a more sustainable solution where it's easier for the project to manage their SVG icons. I see problems with having to replicate the same icon SVG file multiple times to get a range of colors.

Let's discuss a better solution with the team.